samples/csharp/MicrosoftEduGraphSamples/Workflows/AssignmentWorkflow.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Graph;
+using MicrosoftGraphSDK;
+
+namespace MicrosoftEduGraphSamples.Workflows
+{
+    /// <summary>
+    /// Contains all the workflows related to Assignments
+    /// </summary>
+    internal class AssignmentWorkflow
+    {
+        private readonly IConfiguration _config;
+
+        public AssignmentWorkflow(IConfiguration configuration)
+        {
+            this._config = configuration;
+        }
+
+        /// <summary>
+        /// Workflow to get assignments from all the classes which are not archived
+        /// </summary>
+        public void GetNotArchivedClassesAssignments()
+        {
+            try
+            {
+                List<EducationAssignment> assignmentsFromNonArchivedClasses = new List<EducationAssignment>();
```

Comment:
consider: just "assignments" or "allAssignments" (since may or may not be from archived classes)